### PR TITLE
Edited 404 Page

### PR DESCRIPTION
**Pull request by DogeDevYT**
Added css page for 404 to make it look better among other things

### CODE REVIEW

File: 404.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+footer {
+  color: #d6351c;
+  font-family: monospace;
```
Comment (ricoip) [inline 404.css:3]: wouldn't this have no effect since we already wrapped it around `<code>` tags?

File: 404.html [hunk 2]
```diff
@@ -13,13 +13,12 @@
...
-      <!-- Main CSS -->
-        <!-- css gone -->
-      <!-- Overrides -->
+      <link rel="stylesheet" href="404.css">
     <!-- End of CSS -->
   </head>
   <body>
-    <h1>Hey!</h1>
+    <h1>Uh Oh</h1>
+    <img src="media/monkeys/telephonemonkey.gif"><src="media/monkeys/telephonemonkey.gif">
```
Comment (ricoip) [inline 404.html:21]: also why are there two tags lol

**Review by ricoip — CHANGES_REQUESTED** [review]

File: 404.css (hunk 1 above)
Comment (SoNothingMC) [inline 404.css:3]: wouldn't this have no effect on the text since we already wrapped it with `<code>` tags?

File: 404.html (hunk 2 above)
Comment (SoNothingMC) [inline 404.html:21]: why are there two tags lol

this contained the entire page after this in a `<src>` tag

Comment (SoNothingMC) [reply 404.html:21]: (maybe we should add some Alt Text too?)

**Review by SoNothingMC — CHANGES_REQUESTED** [review]

**Comment by DogeDevYT** [comment]
@SoNothingMC I added the alt text and deleted the extra tag.

**Comment by SoNothingMC** [comment]
> 
> 
> @SoNothingMC I added the alt text and deleted the extra tag.

thanks!

**Comment by DogeDevYT** [comment]
@SoNothingMC I deleted the font: monospace tag would it be eligible to merge now?

**Comment by DogeDevYT** [comment]
So would you close this pull request now?

**Comment by SoNothingMC** [comment]
> 
> 
> would it be eligible to merge now?

yee we looking good to merge

**Review by SoNothingMC — APPROVED** [review]
looking gud now

**Comment by SoNothingMC** [comment]
welcome to the contributor club

**Pull request merged**